emacs-travis.mk

```diff
@@ -42,6 +39,17 @@ MAJOR_VERSION := $(word 1,$(subst ., ,$(EMACS_VERSION)))
 # Whether the version is a release candidate
 PRETEST ?= $(findstring rc,$(PRE_RELEASE_PART))
 
+# Build a minimal Emacs with no special flags, to build as fast as possible
+ifndef EMACSCONFFLAGS
```

Comment:
Why the `ifndef` here?  Is it because `+=` is not compatible with `?=`?

Reply:
Yep! `?=` is sugar for `ifndef`, so using the latter gives us some more flexibility.